api/envoy/config/listener/v3/listener.proto

```diff
@@ -390,3 +398,15 @@ message ValidationListenerManager {
 // [#not-implemented-hide:]
 message ApiListenerManager {
 }
+
+// Listener FCDS: In case of inotify/file-system based xDS, collection_name should not be provided
+// This name is only necessary when we are using grpc/REST based xDS, like ADS.
+//[not-implemented-hide:]
+message ListenerFcds{
+  // collection_name indicates the name of the set of filter chains that a listener would subscribe to.
+  // For GRPC based FCDS, this name is mandatory. This name is used to construct a glob collection of filter chains.
+  // For file based FCDS, all the filter chains for a listener are inside the same file.
+  // Hence, there is no need of any group name
+  string collection_name = 1 [(validate.rules).string.min_len = 1];
```

Comment:
The PGV constraint requires a `collection_name`, but this contradicts the comments that state that it should not be provided.

If this is a collection, it may be better to use the [xDS-TP collections](https://github.com/cncf/xds/blob/main/proposals/TP1-xds-transport-next.md#collections-1) notation.

Reply:
Hi @adisuissa I was on paternity. Will work on your comments and get smaller PRs out soon. Thanks.